cmd/traffic_top/Makefile.am

```diff
@@ -17,13 +17,13 @@
 
 # We only build traffic_top if libcurses and libcurl are available.
 if BUILD_TRAFFIC_TOP
-
 AM_CPPFLAGS = \
   $(iocore_include_dirs) \
   -I$(abs_top_srcdir)/lib \
   -I$(abs_top_srcdir)/mgmt/api/include \
   @CURL_CFLAGS@ \
   @CURSES_CFLAGS@
+endif
```

Comment:
Yes, when I added it back in I put it in the wrong place.  I will move it and make sure it works like it is supposed to.